frontend/public/components/utils/dropdown.jsx

```diff
@@ -388,6 +388,8 @@ export class Dropdown extends DropdownMixin {
               onclick={this.onClick}
               selected={ai.actionKey === selectedKey && !noSelection}
               hover={ai.actionKey === keyboardHoverKey}
+              actionItems={actionItems}
+              onChange={this.props.onChange}
```

Comment:
These changes shouldn't be necessary. `this.onClick` will be called whenever an action item is clicked, and I don't see a need to pass all action items into each action item row.
```suggestion
              actionItems={actionItems}
              onChange={this.props.onChange}
```

Reply:
Makes sense. Removed it.